client/state/sites/selectors.js

```diff
@@ -145,7 +170,7 @@ export function isJetpackMinimumVersion( state, siteId, version ) {
  * @return {?String}        Site slug
  */
 export function getSiteSlug( state, siteId ) {
-	const site = getSite( state, siteId );
+	const site = getRawSite( state, siteId );
```

Comment:
Right, but I mean that `getSiteSlug` should just do `return getSite( state, siteId ).slug;` with slug being computed by the new `getSite`.